.tests/cypress/_integration/cart/add-to-cart-bundle.js

```diff
@@ -23,21 +24,13 @@ describe('Bundle product', function() {
   })
 
   it('Test product options', () => {
-    cy.get('#product-options-wrapper')
-      .should('be.visible')
-      .then(elements => {
-        cy.log(elements.length)
-      })
-    cy.get('.radio__fieldset')
-      .find('.radio__label')
-      .click({
-        multiple: true
-      })
-      .then(elements => {
-        cy.log(elements.length)
-      })
+    cy.get('#product-options-wrapper').should('be.visible')
+
+    // TODO: Use better selectors to not use "multiple"
```

Comment:
Yes we agree, but the problem is that the selectors provided from the frontend are not unique, therefore not giving us a choice here. So we can not separate `radio buttons` for each of the product connected to the bundle, we need unique selectors.